app/qml/telescope.qml

```diff
@@ -0,0 +1,86 @@
+import QtQuick 2.4
+import Ubuntu.Components 1.3
+import Telescope 1.0
+
+/*!
+    \brief MainView with a Label and Button elements.
```

Comment:
Remember to remove this comment from the template

Reply:
done